src/align/superposition.ts

```diff
@@ -133,7 +133,7 @@ class Superposition {
     multiplyABt(tmp_1,add,tmp_2)
 
     transpose(transformMat_,tmp_1)
-    this.transformationMatrix.elements = transformMat_.data
+    this.transformationMatrix.elements = Array.from(transformMat_.data)
```

Comment:
If this is made only for avoiding TS complaining, I would suggest to replace this operation by a forced type casting: `transformMat_.data as unknown as number[]`